src/otx/algo/classification/vit.py

```diff
@@ -262,6 +238,15 @@ def __init__(
 
     def load_from_otx_v1_ckpt(self, state_dict: dict, add_prefix: str = "model.") -> dict:
         """Load the previous OTX ckpt according to OTX2.0."""
+        for key in list(state_dict.keys()):
+            new_key = key.replace("patch_embed.projection", "patch_embed.proj")
+            new_key = new_key.replace("backbone.ln1", "backbone.norm")
+            new_key = new_key.replace("ffn.layers.0.0", "mlp.fc1")
+            new_key = new_key.replace("ffn.layers.1", "mlp.fc2")
+            new_key = new_key.replace("layers", "blocks")
+            new_key = new_key.replace("ln", "norm")
+            if new_key != key:
+                state_dict[new_key] = state_dict.pop(key)
         return OTXv1Helper.load_cls_effnet_b0_ckpt(state_dict, "multiclass", add_prefix)
```

Comment:
It seems to be duplicated in other classes, could you consider moving this into `OTXMulticlassClsModel` if there is no issue?

Reply:
Sorry to misunderstand. above codes are only for importing vit pretrained weight provided by mmclassification or otx v1 into a new vit implementation.

Reply:
I know. I just suggested that that function is in other classes equally, I thought it can be handled in one place.
But looking closely, they have different parent classes, `OTXMulticlassClsModel`, `OTXMultilabelClsModel`, and `OTXHlabelClsModel`.
So, if it is possible, how about creating mixin for something related to mmx or v1 legacy?
https://github.com/openvinotoolkit/training_extensions/blob/738fb982cd4678cb6f385b9d17ae916aa9e700c1/src/otx/algo/classification/vit.py#L239-L250
https://github.com/openvinotoolkit/training_extensions/blob/738fb982cd4678cb6f385b9d17ae916aa9e700c1/src/otx/algo/classification/vit.py#L475-L486
https://github.com/openvinotoolkit/training_extensions/blob/738fb982cd4678cb6f385b9d17ae916aa9e700c1/src/otx/algo/classification/vit.py#L620-L631

Reply:
There are multiple redundancies between tasks.. I felt this is required but this looks like refactoring. Let me resolve them in the following PR.